### PR TITLE
Remove ThinEntity

### PR DESCRIPTION
**Pull request by ggeoffrey**

### CODE REVIEW

File: src/hyperfiddle/api.cljc [hunk 1]
```diff
@@ -291,3 +275,21 @@
...
+(defn ->colored-tempid [dbname id]
+  (str "hyperfiddle.tempid-" (str id) "@" dbname))
+
+(defn parse
+  "Parse a given `tempid` to `[db id]`."
+  [id]
+  ;; TODO guard against malformed tempids
+  (when (colored-tempid? id)
+    (-> (str/replace id "hyperfiddle.tempid-" "")
+        (str/split #"@"))))
```
Comment (ggeoffrey) [inline src/hyperfiddle/api.cljc:295]: Oops, forgot about
- malformed tempid's
- parsing efficiency

File: test/hyperfiddle/route_test.cljc [hunk 2]
```diff
@@ -8,27 +8,28 @@
 (defn encode [route] (url-encode route `(foo)))
 (defn decode [s] (url-decode s `(foo)))
 
-(def route-args2 `(hyperfiddle.blog/post ~#entity["$" [:user/sub "google-oauth2|116635422485042503270"]] #{"events" "news"}))
-(def route-args1 `(hyperfiddle.blog/post ~#entity["$" [:user/sub "google-oauth2|116635422485042503270"]]))
-(def route-args1-seq `(hyperfiddle.blog/post ~#entity["$" [:user/sub "google-oauth2|116635422485042503270"]]))
+(def route-args2 `(hyperfiddle.blog/post "$@[:user/sub \"google-oauth2|116635422485042503270\"]" #{"events" "news"}))
```
Comment (ggeoffrey) [inline test/hyperfiddle/route_test.cljc:11]: This is nonsense, I need to fix this file.

File: src/hypercrud/browser/router_bidi.cljc [hunk 3]
```diff
@@ -107,10 +59,7 @@
   (def path-and-frag "/:hyperblog.2!tag/:hyperfiddle")
   (def router ["/"
                {"drafts/" :hyperblog/drafts
-                "pairing/" :user/pairing
-                ; Attempting to call unbound fn: #'hyperfiddle.readers/entity
-                ;[#entity["$" :a]] :hyperblog/post
-                }])
+                "pairing/" :user/pairing}])
```
Comment (dustingetz) [inline src/hypercrud/browser/router_bidi.cljc:62]: the whole file router_bidi.cljc is junk

Comment (dustingetz) [reply src/hypercrud/browser/router_bidi.cljc:62]: also router_bidi_test

Comment (ggeoffrey) [reply src/hypercrud/browser/router_bidi.cljc:62]: Fixed by 488f4bd

**Pull request merged**